aws/dynamodb.go

```diff
@@ -61,6 +59,22 @@ func getAllDynamoTables(session *session.Session, excludeAfter time.Time, db Dyn
 	return tableNames, nil
 }
 
+func shouldIncludeTable(table *dynamodb.TableDescription, excludeAfter time.Time, configObj config.Config) bool{
```

Comment:
`shouldIncludeTable` is not covered by tests from `aws/dynamodb_test.go`

![image](https://user-images.githubusercontent.com/10694338/145541462-d7f0e742-80a6-4a95-b305-04681300753f.png)

[coverage.html.zip](https://github.com/gruntwork-io/cloud-nuke/files/7691179/coverage.html.zip)

Reply:
Added new test for function:

❯ go test -v -run TestShouldIncludeTable
=== RUN   TestShouldIncludeTable
=== RUN   TestShouldIncludeTable/ConfigExclude
=== RUN   TestShouldIncludeTable/ConfigInclude
=== RUN   TestShouldIncludeTable/NotOlderThan
--- PASS: TestShouldIncludeTable (0.00s)
    --- PASS: TestShouldIncludeTable/ConfigExclude (0.00s)
    --- PASS: TestShouldIncludeTable/ConfigInclude (0.00s)
    --- PASS: TestShouldIncludeTable/NotOlderThan (0.00s)
PASS
ok      github.com/gruntwork-io/cloud-nuke/aws  0.201s